packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.types.ts

```diff
@@ -153,6 +153,11 @@ export interface IBreadcrumbItem {
    * Optional role for the breadcrumb item (which renders as a button by default)
    */
   role?: string;
+
+  /**
+   * Any additional properties to apply to the breadcrumb item.
+   */
+  [propertyName: string]: any;
```

Comment:
Rather than adding this (which removes all type safety), if all that's needed is aria attributes, you can make the interface extend `React.AllHTMLAttributes<HTMLElement>`.

Reply:
I was going more for the open door but you are right, what the user needs is to add accessibility props and if provided typified then it's better. Will update 👍🏻